PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,3 +19,35 @@ If this pull request addresses a Help Wanted ticket, please link the relevant Gi
 Otherwise, link the JIRA ticket.
 -->
 
+#### Release Note
+<!--
+Add a release note for each of the following conditions:
+
+* Config changes (additions, deletions, updates)
+* API additions—new endpoint, new response fields, or newly accepted request parameters
+* Database changes (any)
+* Websocket additions or changes
+* Anything noteworthy to a Mattermost instance administrator (err on the side of over-communicating)
+* New features and improvements, including behavioural changes, UI changes and CLI changes
+* Bug fixes and fixes of previous known issues
+* Deprecation warnings, breaking changes, or compatibility notes
+
+If no release notes are required write NONE. Use past-tense. Newlines are stripped.
+
+Examples:
+
+```
+Added new API endpoints POST /api/v4/foo, GET api/v4/foo, and GET api/v4/foo/:foo_id.
+```
+
+```
+Added a new config setting ServiceSettings.FooBar. Added a new column Foo to the Users table.
+```
+
+```
+NONE
+```
+-->
+```release-note
+
+```
```

Comment:
Not every repo is using PR descriptions to fetch the release notes e.g. plugins don't. Should this part of the template only be used by repo that use the release notes automation?